shotover-proxy/tests/redis_int_tests/basic_driver_tests.rs

```diff
@@ -307,8 +307,8 @@ async fn test_filtered_scanning(connection: &mut Connection, flusher: &mut Flush
     let mut unseen = HashSet::<usize>::new();
 
     for x in 0..3000 {
-        let _: () = connection
-            .hset("foo", format!("key_{}_{}", x % 100, x), x)
+        connection
+            .hset::<&'_ str, String, usize, ()>("foo", format!("key_{}_{}", x % 100, x), x)
```

Comment:
Could `<_, _, _, ()>` work too? or maybe it's better to be explicit? 🤔 LGTM as-is.